lib/acquia-http-hmac/rack_authenticate.rb

```diff
@@ -89,11 +89,11 @@ def args_for_authenticator(env, attributes)
 
       def valid_body?(env)
         request = Rack::Request.new(env)
-        if ['GET', 'HEAD'].include?(request.request_method)
+        body = request.body.read   # read the incoming request IO stream
```

Comment:
I'd like to see `request.body.rewind` after this. The Rack interface requires `#rewind` be supported and I'm guessing other users don't call it in advance.